human_analysis/nbconverted/1_process_recount2_data.py

```diff
@@ -0,0 +1,389 @@
+
+# coding: utf-8
+
+# # Process recount2 data
+# This notebook does the following:
+# 
+# 1. Selects template experiment
+# 2. Downloads subset of recount2 data, including the template experiment (50 random experiments + 1 template experiment)
+# 3. Train VAE on subset of recount2 data
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('load_ext', 'rpy2.ipython')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import sys
+import pandas as pd
+import numpy as np
+import random
+import rpy2
+import seaborn as sns
+from sklearn import preprocessing
+import pickle
+
+sys.path.append("../")
+from ponyo import generate_template_data, utils, pipeline
+from functions import process, calc
+
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+# In[2]:
+
+
+# Read in config variables
+base_dir = os.path.abspath(os.path.join(os.getcwd(),"../"))
+
+config_file = os.path.abspath(os.path.join(base_dir,
+                                           "config_human.tsv"))
+params = utils.read_config(config_file)
+
+
+# ### Select template experiment
+# 
+# We manually selected bioproject [SRP012656](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37764), which contains primary non-small cell lung adenocarcinoma tumors and adjacent normal tissues of 6 never-smoker Korean female patients with 2 replicates each.
+
+# In[3]:
+
+
+# Load params
+local_dir = params["local_dir"]
+dataset_name = params['dataset_name']
+NN_architecture = params['NN_architecture']
+project_id = params['project_id']
+
+
+# ### Download subset of recount2 to use as a compendium
+# The compendium will be composed of random experiments + the selected template experiment
+
+# In[4]:
+
+
+get_ipython().run_cell_magic('R', '', '# Select 59\n# Select a\n# Run one time\n#if (!requireNamespace("BiocManager", quietly = TRUE))\n#    install.packages("BiocManager")\n#BiocManager::install("derfinder")\n#BiocManager::install("recount")')
+
```

Comment:
This cell should probably be uncommented for the sake of future users of the notebook

Reply:
Eventually I will need to uncomment this out. For now its commented so that I don't have to re-download a new batch of recount2 data each time i re-run this notebook for testing